partials/55_session_keybindings

```diff
@@ -6,6 +6,10 @@
 set_from_resource $i3-wm.binding.exit_app i3-wm.binding.exit_app Shift+q
 bindsym $mod+$i3-wm.binding.exit_app [con_id="__focused__"] kill
 
+## Session // Kill Parent // <><Ctrl> q ##
```

Comment:
I'm happy to merge this as-is but wondering if there is a more descriptive description.  Also we use the word "Exit" above to indicate that the apps normal exit workflow is called.  IMO "kill" may imply that the process is not cleanly terminated.  Would "Exit Apps in Workspace" be misleading?  Something better?

Reply:
Good point. The problem is workspace is not technically true if you have a split-in-a-split. Say you have a horizontal split and in the right horizontal you then make a vertical two have 3 windows in total, and you call this when in one of the vertically split windows you'll still have the leftmost window open.

On the other hand, I think new users (thinking back to my first days with Regolith) have no idea how child-parent-container things work so writing the technically correct container would not be super helpful.

Maybe "Exit all apps in container (usually workspace)" is not super long but still brings the idea across?

The variable name could be `exit_all` or maybe better `exit_container`.

Reply:
How about `Exit apps in container`?   We can add additional context to what this really does on the site here: https://github.com/regolith-linux/regolith-desktop-website/blob/main/content/docs/using-regolith/configuration.md#all-i3-configuration-packages.  IMO `exit_container` is most consistent.

Reply:
Actually that makes sense. I updated the PR.